package.json

```diff
@@ -6,25 +6,19 @@
   },
   "packageManager": "pnpm@7.14.1",
   "scripts": {
-    "lerna": "lerna",
-    "start": "pnpm watch",
-    "dev": "pnpm start",
-    "test": "pnpm --filter \"@trpc/server\" exec jest",
+    "build": "pnpm codegen && turbo --filter=\"@trpc/*\" build",
+    "dev": "turbo --filter=\"@trpc/*\" dev",
+    "test": "turbo --filter=\"@trpc/tests\" test",
     "lint": "eslint --ext \".js,.ts,.tsx\" --ignore-path .gitignore .",
     "lint-fix": "pnpm lint --fix && manypkg fix",
     "clean": "find . -name node_modules -o -name .turbo -o -name .next -o -name dist -type d -prune | xargs rm -rf",
     "codegen:generateBigBoi": "tsx scripts/generateBigBoi.ts",
     "codegen:generateBigLegacyRouter": "tsx scripts/generateBigLegacyRouter.ts",
-    "codegen:entrypoints": "tsx scripts/entrypoints.ts",
     "codegen:override-prisma": "tsx scripts/addPrismaOverrides.ts",
     "codegen:generateMergeRouters": "tsx scripts/generateMergeRouters.ts",
```

Comment:
I've moved the entry point script into each package, we might be able to separate out the other codegen too so that's cacheable